pyface/ui/__init__.py

```diff
@@ -0,0 +1,84 @@
+# (C) Copyright 2005-2023 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+from importlib import import_module
+from importlib.abc import MetaPathFinder, Loader
+from importlib.util import find_spec
+from importlib.machinery import ModuleSpec
```

Comment:
Swap these around for 🔤 reasons? (Not sure how much we care. `isort` would do something different here anyway, since it likes to put all the `from` imports below the `import` imports in any given section.)

Reply:
Done.